sql/updates/auth/master/2024_08_31_01_auth.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO build_auth_key VALUES (56421, 'Mac', 'A64', 'WoW', 0x5892FFABAEFDCECB0CFFAAA55D2F9B13);
```

Comment:
Add delete like this one

https://github.com/TrinityCore/TrinityCore/commit/cd10553da16bcd99838bd5976939aa314c319b92#diff-a99f5964e5003b34832c13ea6db4f4d02ce121a47dc06c31217e40fd10c9c290R6